templates/taxi/driver_list.html

```diff
@@ -5,13 +5,22 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+        {% if user.is_authenticated and user.id == driver.id%}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }}) (Me)
+          </li>
+
+        {% else %}
+
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }})
+          </li>
+        {% endif %}
```

Comment:
You shouldn't duplicate code. What about this?
```suggestion
       
          <li>
            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
            ({{ driver.first_name }} {{ driver.last_name }})
             {% if user.is_authenticated and user.id == driver.id%}
             (Me)
             {% endif %}
          </li>
```